scripts/dev/e2e.py

```diff
@@ -11,6 +11,9 @@
 import yaml
 
 TEST_POD_NAME = "e2e-test"
+TEST_CLUSTER_ROLE_NAME = "e2e-test"
+TEST_CLUSTER_ROLE_BINDING_NAME = "e2e-test"
+TEST_SERVICE_ACCOUNT_NAME = "e2e-test"
 
```

Comment:
We could summarise those in one constant, eg TEST_NAME -- I kept it as is for clarity, but happy to make changes